working/augmentation-libraries/feature-specification.md

```diff
@@ -176,16 +176,16 @@ members are filled in by an augmentation declaration. _This is primarily useful
 for macros, which may be used to provide a body for an abstract member._
 
 An augmentation that replaces the body of a function, may also want to preserve
-and run the code of the original declaration that it augments (hence the name
-"augmentation"). It may want to run its own code before the original code,
-after it, or both. To support that, we allow a new expression syntax inside the
-“bodies” of augmenting declarations (function bodies, constructor bodies, and
-variable initializers). Inside an expression of a member marked
-`augment`, the identifier `augmented` can be used to refer to the augmented
-function, getter, or setter body, or variable initializer. This is a contextual
-keyword within `augment` members, and has no special meaning outside of that
-context. See the next section for a full specification of what `augmented`
-means, and how it must be used, in the various contexts.
+and run the code of the augmented declaration (hence the name "augmentation").
+It may want to run its own code before the augmented code, after it, or both.
+To support that, we allow a new expression syntax inside the “bodies” of
+augmenting declarations (function bodies, constructor bodies, and variable
+initializers). Inside an expression of a member marked `augment`, the identifier
+`augmented` can be used to refer to the augmented function, getter, or setter
+body, or variable initializer. This is a contextual keyword within `augment`
+members, and has no special meaning outside of that context. See the next
```

Comment:
Nit: "members" -> "declarations".

Since this paragraph isn't the full spec, it doesn't really matter, but using "declarations" here I think helps hint that `augment` is a keyword within an entire augmented class/mixin/etc. even if the member containing `augment` isn't itself an augmentation.